llama_hub/pdf_table/README.md

```diff
@@ -0,0 +1,25 @@
+# PDF Table Loader
+
+This loader reads the tables included in the PDF.
+
+Users can input the PDF `file` and the `pages` from which they want to extract tables, and they can read the tables included on those pages.
+
+## Usage
+
+Here's an example usage of the PDFTableReader.
+`pages` parameter is the same as camelot's `pages`. Therefore, you can use patterns such as `all`, `1,2,3`, `10-20`, and so on.
+
+```python
+from llama_index import download_loader
```

Comment:
nit: could you do `from llama_hub.pdf_table.base import PDFTableReader`

Reply:
Done! thanks :)